models/ListVersion.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const ListVersionSchema = new Schema(
+  {
+    listId: { type: Schema.Types.ObjectId, ref: "List" },
+    text: { type: Schema.Types.ObjectId, ref: "TextString" },
+    month: Number,
```

Comment:
Let's just save a timestamp instead, then we can use only the month & year if day is not relevant.
Since both of these fields are coupled, we should just save the data in once field:
`date: { type: Date, default: Date.now},`